raddb/mods-available/totp

```diff
@@ -43,4 +43,27 @@
 #  The module takes no configuration items.
```

Comment:
that comment should be fixed.  :)  I'll push a change for 3.2

Reply:
Ops! done